app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,7 @@
   </head>
 
   <body>
+    <%= flash[:alert]%>
```

Comment:
In the `LinksController` you use `flash.now[:notice]`, but that message will not be displayed because `<%= flash[:notice] %>` is missing here. I suggest adding it to ensure that the message displays.